From 6f370e318a81b5127206cf8314ad533616495a7d Mon Sep 17 00:00:00 2001
From: Wenping Zhang <wenping.zhang@rock-chips.com>
Date: Fri, 20 Oct 2017 10:17:06 +0800
Subject: [PATCH] ARM: dts: rk3229-gva: change the emmc mode to ddr 52Mhz.

the maximam frequece supported of rk3229 is 125Mhz, so we change
the emmc mode from hs200 to ddr 52Mhz.

Change-Id: I9d6f2a4bc7651cea897f0ff63de571f5492f6c1e
Signed-off-by: Wenping Zhang <wenping.zhang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-gva-sdk.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3229-gva-sdk.dts b/arch/arm/boot/dts/rk3229-gva-sdk.dts
index e9874e3ff6fc..c67a6bba7778 100644
--- a/arch/arm/boot/dts/rk3229-gva-sdk.dts
+++ b/arch/arm/boot/dts/rk3229-gva-sdk.dts
@@ -131,10 +131,9 @@
 };
 
 &emmc {
-	max-frequency = <150000000>;
+	max-frequency = <125000000>;
 	cap-mmc-highspeed;
 	mmc-ddr-1_8v;
-	mmc-hs200-1_8v;
 	non-removable;
 	supports-emmc;
 	status = "okay";
-- 
2.35.3

